drivers/lcd/lcd_dev.c

```diff
@@ -0,0 +1,234 @@
+/****************************************************************************
+ * lcd_dev/lcd_dev.c
+ *
+ *   Copyright (C) 2019 . All rights reserved.
+ *   Author:  <>
```

Comment:
License

Reply:
Strange, I remembered changing that. Just force pushed that.